lib/server_health_check_rails/engine.rb

```diff
@@ -1,5 +1,16 @@
 module ServerHealthCheckRails
   class Engine < ::Rails::Engine
     isolate_namespace ServerHealthCheckRails
+
+    initializer 'server_health_check_rails.bypass_ssl_for_health_index_path' do |app|
```

Comment:
Then I think an extension gem that depends on this gem would be more appropriate. This is a gem published to rubygems.org, and so I believe it should be targeted towards a general audience. I think configuring http vs https and customizing logging would be very surprising to someone interested in easily exposing a health check endpoint.